### DISCUSSION TITLE
How does v6 rotate the canvas or rotate the background?

### DISCUSSION THREAD

**Discussion by Lin-Min** [fabricjs/fabric.js#10367] [Q&A]
I want to rotate the canvas or rotate the background image, and the elements on the canvas will rotate with the rotation that is, the whole rotation. But don't know how to set it

**Comment by asturur** [comment]
What are you trying to achieve? is this rotation interactive?
You can put everything in a group and rotate the group, put the image as the bottom-most object of the canvas instead of using the background image property.
